app/src/main/java/com/celzero/bravedns/util/OrbotHelper.kt

```diff
@@ -286,10 +291,35 @@ class OrbotHelper(private val persistentState: PersistentState, private val prox
         return builder
     }
 
+    private fun fetchColor(context : Context): Int {
```

Comment:
i see this method copy-pasted in multiple files at this point. abstract it out.

Reply:
Removed it from `OrbotHelper` and `BraveVPNService` class.